.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
This file probably shouldn't be committed.

Reply:
i dont' even know how that showed up 😖

Reply:
add `./vscode` to 

https://github.com/SAP/fundamental-react/blob/master/.gitignore